src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -212,6 +216,22 @@ StorageReplicatedMergeTree::StorageReplicatedMergeTree(
     , allow_renaming(allow_renaming_)
     , replicated_fetches_pool_size(global_context.getSettingsRef().background_fetches_pool_size)
 {
+    if (zookeeper_path_.empty())
```

Comment:
Looks like copy-paste. We can have a common function instead.

Reply:
Sure. I fix it soon. Thanks.